docs/shared/recipes/repo-types/package-based-in-integrated.md

```diff
@@ -0,0 +1,45 @@
+# Add a Package-Based Project in an Integrated Repository
+
+An integrated repo offers a lot of features at the cost of some flexibility, but sometimes you want to take back control of your build system or dependency management for a single project in the repo. This recipe shows you how to do that.
+
+A package-based project in an integrated repository is a project that has at least one of these characteristics:
+
+1. Maintains its own dependencies with a separate package.json file
+2. Runs tasks without the use of an Nx plugin
```

Comment:
should we add that it defines tasks in a `package.json` file & doesn't use Nx plugins to run them. Although I almost feel the "doesn't need Nx plugins" to run them is something ppl wouldn't recognize anyway, because in the end you use `nx` to run the task (regardless whether in the package.json or project.json) and then Nx internally calls the plugin's executor etc..